### PR TITLE
samples/bookinfo: update README

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Comment by daixiang0** [comment]
Remove dead link.

**Comment by sdake** [comment]
@daixiang0 thanks for the work, although I don't know this is quite right. While the PR definitely removes a dead link, I think there may be a better way to document how to regenerate the bookinfo manifests. Unfortunately, I don't know how that is done as it is not documented.

Cheers,
-steve

File: samples/bookinfo/README.md [hunk 1]
```diff
@@ -30,6 +30,4 @@ build_push_update_images.sh <version>
 
 ## Tests
 
-Bookinfo is tested by istio.io integration tests. You can find them under [tests/examples](https://github.com/istio/istio.io/tree/master/tests/examples) in the [istio/istio.io](https://github.com/istio/istio.io) repository.
-
-The reference productpage HTML files are in [tests/apps/bookinfo/output](https://github.com/istio/istio/tree/master/tests/apps/bookinfo/output). If the productpage HTML produced by the app is changed, remember to regenerate the reference HTML files and commit them with the same PR.
+Bookinfo is tested by istio.io integration tests. You can find them under [tests](https://github.com/istio/istio.io/tree/master/tests/examples) in the [istio/istio.io](https://github.com/istio/istio.io) repository.
```
Comment (ericvn) [inline samples/bookinfo/README.md:33]:
```suggestion
Bookinfo is tested by istio.io integration tests. You can find them under [tests](https://github.com/istio/istio.io/tree/master/tests) in the [istio/istio.io](https://github.com/istio/istio.io) repository.
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**